test/e2e/sut_helper.go

```diff
@@ -0,0 +1,262 @@
+package e2e
+
+import (
+	"bufio"
+	"container/ring"
+	"context"
+	"fmt"
+	"io"
+	"maps"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"sync"
+	"syscall"
+	"testing"
+	"time"
+
+	"github.com/cometbft/cometbft/p2p"
+	rpchttp "github.com/cometbft/cometbft/rpc/client/http"
+	"github.com/libp2p/go-libp2p/core/peer"
+	"github.com/rollkit/rollkit/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var WorkDir = "."
+
+type SystemUnderTest struct {
```

Comment:
This is a "light" version of the [system test framework](https://github.com/cosmos/cosmos-sdk/tree/main/systemtests) in cosmos-sdk